src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/Monitor.java

```diff
@@ -120,7 +119,7 @@ public void run() {
     }
   }
 
-  ConnectionStatus isConnectionHealthy(final int shortestFailureDetectionIntervalMillis) {
+  ConnectionStatus getConnectionHealthStatus(final int shortestFailureDetectionIntervalMillis) {
```

Comment:
NIT: Since the object returned is ConnectionStatus, could that be getConnectionStatus? Or are you fetching just things related to the health of the connection?